charts/gateway-helm/templates/envoy-gateway-deployment.yaml

```diff
@@ -1,25 +1,3 @@
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: envoy-gateway
-  namespace: '{{ .Release.Namespace }}'
-  labels:
-  {{- include "eg.labels" . | nindent 4 }}
----
-{{- if and .Values.podDisruptionBudget.minAvailable (ge (int .Values.podDisruptionBudget.minAvailable) 1) }}
```

Comment:
prefer keeping this as this to make this backwards compatible and only inserting
```
maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
```

along with the `if` that will needed to ensure this template kicks in
cc @alexwo @guydc

Reply:
Do you mean you'd prefer having it always being applied, and if not input provided would default to minAvailable=0?

Reply:
the top level `if` could we tweaked to `{{- if and .Values.podDisruptionBudget }}` i.e. create the resource, if any field is set

Reply:
Makes sense! I can adjust it a bit more if preferred